models/enterprise_sonic/mgmt_servers/sonic_mgmt_servers.yaml

```
@@ -71,6 +71,12 @@ DOCUMENTATION: |
                 type: str
                 choices:
                   - mgmt
+              cipher_suite:
+                description:
+                  - Cipher suites used for TLS connection with the clients
+                  - Specify as a comma separated list. Options are ecdhe-ecdsa-with-aes-256-gcm-SHA384, ecdhe-ecdsa-with-chacha20-poly1305-SHA256 and ecdhe-ecdsa-with-aes-128-gcm-SHA256.
```

This line is longer than desirable. Please break it up into smaller lines:

```suggestion
                  - Specify as a comma separated list. Options are ecdhe-ecdsa-with-aes-256-gcm-SHA384,
                  -   ecdhe-ecdsa-with-chacha20-poly1305-SHA256 and
                  -   ecdhe-ecdsa-with-aes-128-gcm-SHA256.
```